tsconfig.json

```diff
@@ -9,9 +9,15 @@
     "outDir": "dist",
     "rootDir": "src",
     "sourceMap": true,
+    "strictNullChecks": true,
     "target": "ES2017",
     "typeRoots": ["./node_modules/@types"]
   },
-  "include": ["./src/**/*.ts"],
-  "exclude": ["**/*.test.ts"]
```

Comment:
Try running `npm publish --dry-run` - it shows that the npmignore file is not working (presumably because `files` is set and that takes precedence).

You could introduce the pattern from the module template where we have a separate `tsconfig.build.json` for the production build which excludes tests, but leave tests included in the top-level one.

Or if you'd prefer, we could update `files` to use a glob that excludes tests

Reply:
Looks like this is still a problem; tests are being included in the package output